wavefront/server/modules/voice_agents_module/voice_agents_module/services/tool_service.py

```
@@ -0,0 +1,574 @@
+            # Invalidate list cache
+            self.cache_manager.remove(self._get_tools_list_cache_key())
+
+            logger.info(f'Created tool: {created_tool.name} (ID: {created_tool.id})')
+            return created_tool
+
+        except Exception as e:
+            logger.error(f'Error creating tool: {str(e)}')
+            raise
+
+    async def get_tool(
+        self, tool_id: UUID, exclude_sensitive: bool = True
```

Ruff (0.14.13)

Unused method argument: `exclude_sensitive`

(ARG002)